Tasks/JenkinsDownloadArtifacts/ArtifactDetails/ArtifactDetailsDownloader.ts

```diff
@@ -0,0 +1,125 @@
+import * as Q from 'q';
+import * as  tl from 'vsts-task-lib/task';
+
+import {JenkinsRestClient} from "./JenkinsRestClient"
+import {CommitsDownloader} from "./CommitsDownloader"
+import {WorkItemsDownloader} from "./WorkItemsDownloader"
+
+var handlebars = require('handlebars');
+
+export class ArtifactDetailsDownloader {
+    public DownloadCommitsAndWorkItems(): Q.Promise<void> {
+        let defer: Q.Deferred<any> = Q.defer<any>();
+
+        console.log(tl.loc("DownloadingCommitsAndWorkItems"));
+        
+        let startBuildId: number = parseInt(tl.getInput("startJenkinsBuildNumber"))
+        let commitsDownloader: CommitsDownloader = new CommitsDownloader();
+
+        if (!startBuildId) {
```

Comment:
It will be good to handle NaN explicitly

Reply:
Fixed.